opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/magma/variablestoview/presenter/VariablesToViewPresenter.java

```diff
@@ -78,6 +78,10 @@ public void saveVariable() {
     getView().clearErrors();
 
     VariableCopyValidationHandler variableCopyValidationHandler = new VariableCopyValidationHandler();
+
+    variableCopyValidationHandler.getValidators().add(new RequiredTextValidator(getView().getViewName(), "NameIsRequired", FormField.NAME.name()));
+    variableCopyValidationHandler.getValidators().add(new RegExValidator(getView().getViewName(), "^[a-zA-Z0-9_-]*$", "NameHasInvalidCharacters", FormField.NAME.name()));
```

Comment:
That is not the same validation as for the creation of a view. Please ensure consistency.

Reply:
used DisallowedCharactersValidator with "ViewNameDisallowedChars" message key and "ViewNameRequired" key for the required text validator